examples/complete/main.tf

```diff
@@ -2,7 +2,12 @@ provider "aws" {
   region = local.region
 }
 
-data "aws_availability_zones" "available" {}
+data "aws_availability_zones" "available" {
+  filter {
```

Comment:
can we add a note so folks know why we do this https://github.com/terraform-aws-modules/terraform-aws-eks/blob/c41b58277ab3951eca8d11863edf178135ec7654/examples/eks-auto-mode/main.tf#L6